cli/client.go

```diff
@@ -1114,6 +1116,29 @@ var clientRetrieveCmd = &cli.Command{
 	},
 }
 
+var clientInspectDealCmd = &cli.Command{
+	Name:  "inspect-deal",
+	Usage: "Inspect storage deal",
```

Comment:
Let's add some description here so the user knows they can pass either deal-id or proposal-cid